### PR TITLE
Update hash-rules.md

### PR DESCRIPTION
**Pull request by stacykor**
Hash 256 updates.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/collection/processing-rules/hash-rules.md [hunk 1]
```diff
@@ -18,17 +16,18 @@ Log line:
...
+[module=RAW] [logger=scala.raw.InboundRawProtocolHandler] [memberid=906e9cc124c8e1085b10e1cec4cc6526f3637558be361d3b4bb54bb537e49a49] [remote_ip=98.248.40.103]
+[web_session=19zefhqy...] [session=80F1BD83AEBDF4FB] [customer=0000000000000005] [call=InboundRawProtocol.getMessages]
 ```
 
 Note the following:
 
 * Values that you want hashed must be expressed as a match group enclosed in "( )".
 * You can use an anchor to detect specific values. In addition, you can specify multiple match groups. If multiple match groups are specified, each of the values will be hashed uniquely.
-* The hash algorithm is MD5.
+* The hash algorithm is MD5 (default) or 256. Open Telemetry collectors only support 256.
```
Comment (kimsauce) [inline docs/send-data/collection/processing-rules/hash-rules.md:30]:
```suggestion
* The hash algorithm is MD5 (default) or 256. OpenTelemetry collectors only support 256.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor change!

**Comment by stacykor** [comment]
Disabling Auto merge. Waiting on Maise

**Comment by stacykor** [comment]
Team decided to wait until Monday.

**Review by MariePMcGarry — APPROVED** [review]

**Pull request closed without merging**